src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -583,6 +583,10 @@ class Testcase(Model):
   # Tracks if a testcase is stuck during triage.
   stuck_in_triage = ndb.BooleanProperty(default=False)
 
+  # Tracks if analyze task is pending.
```

Comment:
Do we want to track if just analyze is pending and not analyze/minimize/regressoin?

Reply:
Minimize/regression/impact are tracked together under critical tasks. The idea for this, is so we would be able to catch that pubsub incident.

Since we already have quite a few stuck testcases under critical tasks, we do not gain much by splitting until we clean up the old stuff.